container/karaf/src/main/filtered-resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -66,6 +66,10 @@ log4j2.logger.kafkaAdminClientConfig.level = ERROR
 log4j2.logger.opennmsTopology.name = org.opennms.features.topology
 log4j2.logger.opennmsTopology.level = WARN
 
+# OPENNMS: Reduce container bridge logging to INFO
+log4j2.logger.opennms.name = org.opennms.container.web.bridge.rest
```

Comment:
I'm not sure about the semantics of this config file, but is it correct to have the same keys (`log4j2.logger.opennms.name` and `level`) as on lines 74-75 below?

Reply:
you are right, fixed!